### PR TITLE
Fct 197 categories presets

### PR DESCRIPTION
**Pull request by tylermorrisford**
This PR creates `category-draft` presets for the Fashion Sample Data.

### CODE REVIEW

File: models/category/src/category-draft/presets/sample-data-fashion/bottoms-kids.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import * as CategoryDraft from '../..';
+import { TCategoryDraftBuilder } from '../../../types';
+
+const bottomsKids = (): TCategoryDraftBuilder =>
+  CategoryDraft.presets
+    .empty()
+    .name(LocalizedString.presets.empty()['en-US']('Bottoms'))
+    .slug(LocalizedString.presets.empty()['en-US']('bottoms_kids'))
+    .description(LocalizedString.presets.empty()['en-US']('Bottoms'))
+    .parent(Reference.presets.category()['key']('clothing-kids'))
```
Comment (stephsprinkle) [inline models/category/src/category-draft/presets/sample-data-fashion/bottoms-kids.ts:11]: Here and elsewhere...
```suggestion
    .parent(Reference.presets.category().key('clothing-kids'))
```

Comment (stephsprinkle) [reply models/category/src/category-draft/presets/sample-data-fashion/bottoms-kids.ts:11]: The parent keys should be pulled in from the appropriate preset and referenced as variables so there aren't any magic strings hanging around.

File: models/category/src/category-draft/presets/sample-data-fashion/bottoms-kids.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    .key('bottoms_kids')
+    .orderHint('0.3')
+    .assets([]);
```
[the remark this answers is not in the document]
Comment (tylermorrisford) [reply models/category/src/category-draft/presets/sample-data-fashion/bottoms-kids.ts:14]: Each of these fields, or just assets?

[the remark this answers is not in the document]
Comment (stephsprinkle) [reply models/category/src/category-draft/presets/sample-data-fashion/bottoms-kids.ts:14]: Just assets

**Review by stephsprinkle — APPROVED** [review]

File: models/category/src/category-draft/presets/empty.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import type { TCategoryDraftBuilder } from '../../types';
+import Category from '../builder';
+
+const empty = (): TCategoryDraftBuilder =>
+  Category()
```
Comment (stephsprinkle) [inline models/category/src/category-draft/presets/empty.ts:5]: Nit: rename this to `CategoryDraft` 😉

Comment (tylermorrisford) [reply models/category/src/category-draft/presets/empty.ts]: 🕺

File: models/category/src/category-draft/generator.ts [hunk 4]
```diff
@@ -11,7 +17,7 @@ const generator = Generator<TCategoryDraft>({
     name: fake(() => LocalizedString.random()),
     slug: fake(() => LocalizedString.presets.ofSlugs()),
     description: null,
-    parent: null,
+    parent: fake(() => KeyReference.random().key(KEY).typeId(CATEGORY)),
```
Comment (jaikamat) [inline models/category/src/category-draft/generator.ts:20]: Fwiw, [this pr](https://github.com/commercetools/test-data/pull/237/files) is now merged and you have access to CategoryReference in KeyReference presets!

**Review by jaikamat — APPROVED** [review]
Turtles all the way down!

**Pull request merged**